packages/nvidia_nat_weave/src/nat/plugins/weave/weave_exporter.py

```diff
@@ -176,6 +188,41 @@ def _create_weave_call(self, step: IntermediateStep, span: Span) -> Call:
 
         return call
 
+    def _extract_output_message(self, output_data: Any, outputs: dict[str, Any]) -> None:
+        """
+        Extract message content from various response formats and add to outputs dictionary.
+
```

Comment:
Maybe add a note here of the supported formats and indicate that it does not add anything to `outputs` if passed data is not supported?

I had to trace and follow the entire code of this function to understand the effects of what would happen if the data wasn't understood.